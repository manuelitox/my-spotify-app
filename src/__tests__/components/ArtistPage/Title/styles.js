import React from 'react'
import { shallow } from 'enzyme'

import StyledTitle from 'components/ArtistPage/Title/styles'

describe('Component | StyledTitle', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledTitle />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
    expect(component.props().color).toEqual('white')
  })
  it('should returns green color', () => {
    component = shallow(<StyledTitle color="green" />)
    expect(component.props().color).toEqual('green')
  })
})