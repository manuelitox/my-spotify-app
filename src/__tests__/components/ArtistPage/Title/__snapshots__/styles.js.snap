// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledTitle should render a styled component 1`] = `
<StyledComponent
  color="white"
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  color: ",
          [Function],
          ";
  font-size: 2rem;
  font-weight: 500;
  margin: 0;
",
        ],
      },
      "defaultProps": Object {
        "color": "white",
      },
      "displayName": "styled.h2",
      "foldedComponentIds": Array [],
      "propTypes": Object {
        "color": [Function],
      },
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h2",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
